numpyro/hmc_util.py

```diff
@@ -372,16 +375,17 @@ def _combine_tree(current_tree, new_tree, inverse_mass_matrix, going_right, rng,
                      sum_accept_probs, num_proposals)
 
 
-@partial(jit, static_argnums=(0, 1, 10))
-def _build_basetree(vv_update, kinetic_fn, z, r, z_grad, step_size, going_right,
+@partial(jit, static_argnums=(0, 1, 11))
+def _build_basetree(vv_update, kinetic_fn, inverse_mass_matrix, z, r, z_grad, step_size, going_right,
```

Comment:
I think it is better to move this before `step_size` for consistence.

Reply:
Sure, I grouped it so that it is close to `kinetic_fn`, but I'm fine with any consistent grouping. Will make the change.